pyproject.toml

```diff
@@ -0,0 +1,39 @@
+[build-system]
+requires = ["setuptools>=58.1.0", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[tool.pytest.ini_options]
+addopts = "--cov=data_clusterer"
+testpaths = [
+    "src/tests",
+]
+
+[tool.mypy]
```

Comment:
please add configuration of pre-commit hooks together with pre-commit settings (in one commit)